packages/docusaurus/src/client/serverEntry.tsx

```diff
@@ -10,7 +10,6 @@ import path from 'path';
 import fs from 'fs-extra';
 // eslint-disable-next-line no-restricted-imports
 import _ from 'lodash';
-import chalk from 'chalk';
```

Comment:
@Josh-Cena had to remove usage of Chalk in this field, it simply does not work for me and I think it may have never worked actually? It's been a while I see weird error messages related to chalk being reported

Reply:
That's true :/ We can switch back once we are using ESM in SSR. For now let's get rid of it completely unless I have time to investigate in the near future.